package.json

```
@@ -0,0 +1,43 @@
+{
+  "name": "forta-bot-deployed-agent",
+  "version": "0.0.1",
+  "description": "Forta Bot that monitors forta bot deployments from the deployer address.",
+  "repository": "",
```

```suggestion
  "repository": "https://github.com/HansBhatia/Forta-Bot-Deployed-Agent",
```
this should be the repo that contains this bot. in this case it's your repo.